commands/license_info

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+help() {
+  cat <<EOF
+
+----------------------------------------------------------------------------------------------------------
+HELP MESSAGE:
+
+    Usage: c8y oee $(basename "$0") [--view <json|table>]
+
+    Arguments:
+      -v, --view      print-format      The printing format for data (default: table)
```

Comment:
- I think we should align with the common argument names. For the format I'd stick to `-o, --output`: https://goc8ycli.netlify.app/docs/concepts/common-parameters/#output

- the c8y cli also supports `csv` & `csvheader`; could we add that as well?